src/Symfony/Component/DomCrawler/Crawler.php

```diff
@@ -1258,6 +1249,49 @@ private function isValidHtml5Heading(string $heading): bool
         return 1 === preg_match('/^\x{FEFF}?\s*(<!--[^>]*?-->\s*)*$/u', $heading);
     }
 
+    private function copyFromHtml5ToDom(\Dom\Node $source, \DOMDocument $target): void
+    {
+        /** @var list<array{0: iterable<\Dom\Node>, 1: ?\DOMNode}> $stack */
```

Comment:
can the key `1` really be nullable ? The code does not handle null values when using it.